docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -1,28 +1,35 @@
 description: Hyperledger Besu IBFT 2.0 Proof-of-Authority (PoA) consensus protocol implementation
 <!--- END of page meta data -->
 
-*[Vanity]: Validators can include anything they like as vanity data. 
-*[RLP]: Recursive Length Prefix
-*[Byzantine fault tolerant]: Ability to function correctly and reach consensus despite nodes failing or propagating incorrect information to peers.
+*[Vanity]: Validators can include anything they like as vanity data.
+*[RLP]: Recursive Length Prefix.
 
 # IBFT 2.0
```

Comment:
> ./docs/HowTo/Configure/Consensus-Protocols/IBFT.md:1: MD041 First line in file should be a top level header

To prevent this you have to convert file header html style comment to dashes syntax:

```
---
title: the page title
description: the page description
---

# Your first level title
```

Reply:
What about the acronym definitions? Should they be inside the dashes too? I'll experiment.

Reply:
I don't think I can put them inside the dashes. They'll just become comments then. How about at the end of the file?

Reply:
Acronyms and Definitions section added to the end of the file, and definitions included there.